Utils.ps1

```diff
@@ -22,6 +22,9 @@ Set-Alias ?? Invoke-NullCoalescing -Force
 function Invoke-Utf8ConsoleCommand([ScriptBlock]$cmd) {
     $currentEncoding = [Console]::OutputEncoding
     try {
+        # A native executable that writes to stderr AND has its stderr redirected will generate non-terminating
+        # error records if the user has set $ErrorActionPreference to Stop. Override that value in this scope.
+        $ErrorActionPreference = 'Continue'

```

Comment:
Ironically it is the act of redirecting stderr to $null that causes this.  Try it without the redirect and you'll see no updates to $Error.  I'll look into this some more though.  

The behavior of stderr also differs between hosts like PowerShell.exe and PowerShell_ISE.exe.  In the former, the native app is given the console handles for stdout/err/in.